apps/codeimage/src/core/modules/pagination/createPagedData.ts

```diff
@@ -1,24 +1,23 @@
-import {Accessor, createSignal, Setter, createEffect} from 'solid-js';
+import {Accessor, createSignal, Setter} from 'solid-js';
+interface CreatePaginatedProps<T> {
+  data: Accessor<T[] | undefined>;
+  options: {pageSize: number; pageSelected: number};
+}
 
-export const createPagedData = <T>(
-  data: Accessor<T[] | undefined>,
-): [
+export const createPagedData = <T>({
+  data,
+  options,
+}: CreatePaginatedProps<T>): [
   Accessor<T[]>,
-  {page: Accessor<number>; setPage: Setter<number>; pageSize: number},
+  {page: Accessor<number>; setPage: Setter<number>},
 ] => {
-  const [page, setPage] = createSignal(1),
-    pageSize = 9;
-  createEffect(() =>
-    console.log('page', {
-      start: (page() - 1) * pageSize,
-      end: (page() - 1) * pageSize + pageSize,
-    }),
-  );
+  console.log('pageSelected', options.pageSelected);
+  const [page, setPage] = createSignal(options.pageSelected);
```

Comment:
@riccardoperra in the dashboard.state I pass as `options.pageSelected = 5`. In this console.log I've obtained 5 but the `page()` of the dashboard it's still 1 do you know why?